ImageInfo/README.MD

```diff
@@ -9,8 +9,8 @@ Run `.scriptload ImageInfo.js` to load the script. Any module accessed via `dx`,
 This extension extends the debugger's concept of a module to include:
 * <b>Headers</b> - A list of available headers from the PE image for the module.
     * <b>ImageHeader</b> - The DOS header for the module -- an _IMAGE_DOS_HEADER.
-    * <b>FileHeader</b> - The file header for the module -- an [_IMAGE_FILE_HEADER](https://docs.microsoft.com/en-us/windows/desktop/api/winnt/ns-winnt-_image_file_header).
-    * <b>OptionalHeader</b> - The optional header for the module -- an [_IMAGE_OPTIONAL_HEADER(64)](https://docs.microsoft.com/en-us/windows/desktop/api/winnt/ns-winnt-_image_optional_header64).
+    * <b>FileHeader</b> - The file header for the module -- an [_IMAGE_FILE_HEADER](https://docs.microsoft.com/windows/desktop/api/winnt/ns-winnt-_image_file_header).
+    * <b>OptionalHeader</b> - The optional header for the module -- an [_IMAGE_OPTIONAL_HEADER(64)](https://docs.microsoft.com/windows/desktop/api/winnt/ns-winnt-_image_optional_header64).
```

Comment:
these links need the en-us, the new ones are broken

Reply:
Actually, they truly are broken. "en-us" is not related to that.